docs/real-time-analytics/logistics/process.sql

```diff
@@ -0,0 +1,81 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create Vehicle Stream
+CREATE STREAM VEHICLES ( 
+	ID VARCHAR KEY,
+  	ORDER_ID VARCHAR,
+  	STATE VARCHAR,
+  	LAT DOUBLE,
+  	LONG DOUBLE,
+  	DEST_LAT DOUBLE,
+  	DEST_LONG DOUBLE,
+  	WAREHOUSE_LAT DOUBLE,
+ 	WAREHOUSE_LONG DOUBLE,
+  	TEMPERATURE DOUBLE,
+  	TIRE_PRESSURE DOUBLE
+) WITH (
+	KAFKA_TOPIC='fleet_updates', 
+	VALUE_FORMAT='JSON', 
+	KEY_FORMAT='KAFKA',
+	PARTITIONS=4
+);
+
+-- Create Table of Orders
+CREATE TABLE ORDERS (
+	ID VARCHAR PRIMARY KEY, 
+	NAME VARCHAR, 
+	EMAIL VARCHAR, 
+	ADDRESS VARCHAR, 
+	ORDER VARCHAR, 
+	ORDER_TOTAL DOUBLE 
+) WITH (
+	KAFKA_TOPIC='orders', 
+	VALUE_FORMAT='JSON', 
+	KEY_FORMAT='KAFKA',
+	PARTITIONS=4
+);
+
+-- Mask customer data in Orders Table
+CREATE TABLE ORDERS_MASKED WITH (
+	KAFKA_TOPIC='orders_masked', 
+	PARTITIONS=4, 
+	REPLICAS=3
+) AS 
+SELECT
+  	ID ID,
+  	MASK(NAME) MASKED_NAME,
+  	MASK(EMAIL) MASKED_EMAIL,
+  	MASK(ADDRESS) MASKED_ADDRESS,
```

Comment:
None of the masked fields are used in the second query. What is the purpose of this step?

Also, both queries could be merged into a single one.

Reply:
These steps were taken from an old demo. Masking was done presumably to show off the fact that it could be done. I'll remove.